# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1468499459 -7200
#      Thu Jul 14 14:30:59 2016 +0200
# Node ID c8abb2ba40175316b7c2038492730969df8364bc
# Parent  3f5f1234c619c4628270ed9ee5de93e633137606
doc: Fix 2 typos: Seperately -> Separately, lit -> list

diff --git a/doc/interpreter/plot.txi b/doc/interpreter/plot.txi
--- a/doc/interpreter/plot.txi
+++ b/doc/interpreter/plot.txi
@@ -1223,17 +1223,17 @@ A light object used for lighting effects
 @subsubsection Creating Graphics Objects
 @cindex creating graphics objects
 
 You can create any graphics object primitive by calling the function of the
 same name as the object; In other words, @code{figure}, @code{axes},
 @code{line}, @code{text}, @code{image}, @code{patch}, @code{surface}, and
 @code{light} functions.  These fundamental graphic objects automatically become
 children of the current axes object as if @code{hold on} was in place.
-Seperately, axes will automatically become children of the current figure
+Separately, axes will automatically become children of the current figure
 object and figures will become children of the root object 0.
 
 If this auto-joining feature is not desired then it is important to call
 @code{newplot} first to prepare a new figure and axes for plotting.
 Alternatively, the easier way is to call a high-level graphics routine which
 will both create the plot and then populate it with low-level graphics objects.
 Instead of calling @code{line}, use @code{plot}.  Or use @code{surf} instead of
 @code{surface}.  Or use @code{fill} instead of @code{patch}.
diff --git a/scripts/gui/uicontrol.m b/scripts/gui/uicontrol.m
--- a/scripts/gui/uicontrol.m
+++ b/scripts/gui/uicontrol.m
@@ -47,17 +47,17 @@
 ## @item @qcode{"checkbox"}
 ## Create a checkbox control that allows user on/off selection.
 ##
 ## @item @qcode{"edit"}
 ## Create an edit control that allows user input of single or multiple lines
 ## of text.
 ##
 ## @item @qcode{"listbox"}
-## Create a listbox control that displays a lit of items and allows user
+## Create a listbox control that displays a list of items and allows user
 ## selection of single or multiple items.
 ##
 ## @item @qcode{"popupmenu"}
 ## Create a popupmenu control that displays a list of options that can be
 ## selected when the user clicks on the control.
 ##
 ## @item @qcode{"pushbutton"}
 ## Create a push button control that allows user to press to cause an action.
